packages/fxa-payments-server/src/components/PaymentConfirmation/index.stories.tsx

```diff
@@ -83,73 +88,93 @@ const coupon: CouponDetails = {
   maximallyRedeemed: false,
 };
 
-storiesOf('components/PaymentConfirmation', module)
-  .add('default', () => (
-    <MockApp>
-      <PaymentConfirmation
-        {...{ profile: userProfile, selectedPlan, customer, productUrl }}
-      />
-    </MockApp>
-  ))
-  .add('custom action button label', () => (
-    <MockApp>
-      <PaymentConfirmation
-        {...{
-          profile: userProfile,
-          selectedPlan: selectedPlanWithMetadata,
-          customer,
-          productUrl,
-        }}
-      />
-    </MockApp>
-  ))
-  .add('custom action button label localized to xx-pirate', () => (
-    <MockApp languages={['xx-pirate']}>
-      <PaymentConfirmation
-        {...{
-          profile: userProfile,
-          selectedPlan: selectedPlanWithMetadata,
-          customer,
-          productUrl,
-        }}
-      />
-    </MockApp>
-  ))
-  .add('paypal', () => (
-    <MockApp>
-      <PaymentConfirmation
-        {...{
-          profile: userProfile,
-          selectedPlan,
-          customer: PAYPAL_CUSTOMER,
-          productUrl,
-        }}
-      />
-    </MockApp>
-  ))
-  .add('with passwordless account', () => (
-    <MockApp>
-      <PaymentConfirmation
-        {...{
-          profile: userProfile,
-          selectedPlan,
-          customer: PAYPAL_CUSTOMER,
-          productUrl,
-          accountExists: false,
-        }}
-      />
-    </MockApp>
-  ))
-  .add('with coupon', () => (
-    <MockApp>
+type CouponDetailsProps = {
+  profile: Profile;
+  selectedPlan: Plan;
+  customer: Customer;
+  productUrl: string;
+  accountExists?: boolean;
+  coupon?: CouponDetails;
+};
```

Comment:
Q: Would it be better to export the `PaymentConfirmationProps` type from index.tsx versus recreating here (with a different name and mildly different order):
https://github.com/mozilla/fxa/blob/973b1685d063e7e1321896686cb2181470e9c892/packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx#L20-L27

Reply:
Agreed. Code updated!